tests/main/grid/test_eta.py

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env python3
```

Comment:
Please add a description of test with what it is testing, success and/or failure modes, and expected results for success vs failure.